gcc/config/arc64/atomic.md

```diff
@@ -91,7 +91,7 @@
 	(unspec_volatile:SDI
 	  [(match_operand:SDI 1 "mem_noofs_operand" "ATOMC")]
 	  ARC64_VUNSPEC_LL))]
-  ""
+  "TARGET_ATOMIC"
```

Comment:
This has to be be conditional. The 64 bit llockd is only valid from `ATOMIC_2+`.